Fix build failure.
/usr/lib/python3.11/site-packages/setuptools/config/_apply_pyprojecttoml.py:75: _MissingDynamic: `license` defined outside of `pyproject.toml` is ignored.

--- sexpdata-1.0.1/setup.py
+++ sexpdata-1.0.1/setup.py
@@ -12,10 +12,8 @@
     author='Joshua D. Boyd, Takafumi Arakaki',
     author_email='jdboyd@jdboyd.net',
     url='https://github.com/jd-boyd/sexpdata',
-    license='BSD License',
     description='S-expression parser for Python',
     long_description=long_description,
-    keywords='s-expression, lisp, parser',
     classifiers=[
         "Development Status :: 3 - Alpha",
         'License :: OSI Approved :: BSD License',
